src/status_im2/subs/home.cljs

```diff
@@ -1,38 +1,20 @@
 (ns status-im2.subs.home
```

Comment:
Maybe we should rename home namespace to chats or some other more informative name (like chats etc.)